File: Beecrowd/Beginner/1000 - 1099/1094 - Experiments/C++/1094_Experiments.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int number, total = 0, totalCoelho = 0, totalRato = 0, totalSapo = 0;
    cin >> number;

    while (number--) {
        int amount;
        char type;
        cin >> amount >> type;

        total += amount;

        if (type == 'C') totalCoelho += amount;
        else if (type == 'R') totalRato += amount;
        else if (type == 'S') totalSapo += amount;
    }

    cout << "Total: " << total << " cobaias" << endl;
    cout << "Total de coelhos: " << totalCoelho << endl;
    cout << "Total de ratos: " << totalRato << endl;
    cout << "Total de sapos: " << totalSapo << endl;
    cout << fixed << setprecision(2);
    cout << "Percentual de coelhos: " << (double) totalCoelho / total * 100 << " %" << endl;
    cout << "Percentual de ratos: " << (double) totalRato / total * 100 << " %" << endl;
    cout << "Percentual de sapos: " << (double) totalSapo / total * 100 << " %" << endl;
    
    return 0;
}

File: Beecrowd/1 - Beginner/1000 - 1099/1094 - Experiments/1094_Experiments.py
number = int(input())
total = 0
total_coelho = 0
total_rato = 0
total_sapo = 0

for _ in range(number):
    amount, animal_type = input().split()
    amount = int(amount)

    total += amount

    if animal_type == 'C':
        total_coelho += amount
    elif animal_type == 'R':
        total_rato += amount
    elif animal_type == 'S':
        total_sapo += amount

print(f"Total: {total} cobaias")
print(f"Total de coelhos: {total_coelho}")
print(f"Total de ratos: {total_rato}")
print(f"Total de sapos: {total_sapo}")
print(f"Percentual de coelhos: {(total_coelho / total * 100):.2f} %")
print(f"Percentual de ratos: {(total_rato / total * 100):.2f} %")
print(f"Percentual de sapos: {(total_sapo / total * 100):.2f} %")
